src/main/java/com/growup/todoary/service/TodoService.java

```diff
@@ -0,0 +1,19 @@
+package com.growup.todoary.service;
+
+import com.growup.todoary.domain.Todo;
+import com.growup.todoary.repository.TodoRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class TodoService {
+
+    private final TodoRepository todoRepository;
+
+    public List<Todo> getTodos() {
+        return todoRepository.getTodos();
+    }
```

Comment:
위에서 이야기한 JPARepository를 이용한 형태로 변경하면 다음과 같이 바뀌겠군요

```suggestion
    public List<Todo> findAll() {
        return todoRepository.findAll();
    }
```

Reply:
넵 JpaRepository를 사용하는 방법으로 바꾸겠습니다